packages/decorators/src/state/application-name.ts

```diff
@@ -1,24 +1,6 @@
-import {
-    PlatformServiceName,
-    StateOptions,
-} from '@limetech/lime-web-components-interfaces';
-import { createStateDecorator, StateDecoratorConfig } from '../factory';
+import { SelectApplicationName } from '../../../lime-web-components/src';
 
 /**
- * Get the name of the application
- *
- * @param {StateOptions} [options] options for the state selector
- * @returns {Function} state decorator
+ * @deprecated use {@link SelectApplicationName} from `@limetech/lime-web-components` instead
  */
-export function ApplicationName(options: StateOptions = {}) {
-    const config: StateDecoratorConfig = {
-        name: PlatformServiceName.ApplicationState,
-    };
-    options.map = [getApplicationName, ...(options.map || [])];
-
-    return createStateDecorator(options, config);
-}
-
-function getApplicationName(applicationData: any) {
-    return applicationData.applicationName;
-}
+export const ApplicationName = SelectApplicationName;
```

Comment:
For things like this, where we're actually exporting a function, not just an interface, instead of just returning the moved function, I think we could wrap it in a function that logs a deprecation warning, and then returns the result of the moved function… 🤔

```suggestion
export const ApplicationName = (...args) => {
	console.warn('@limetech/lime-web-components-decorators has been deprecated. Please import the ApplicationName decorator from @limetech/lime-web-components instead.')
	return SelectApplicationName(...args);
};
```

Thoughts?

Reply:
I tried it in the console, and it seems to work just fine:

<img width="1437" alt="A screenshot of the dev-tools console, where I have created a simple function with three arguments, then wrapped that in a function like the one suggested in the comment above, along with the result showing that it works." src="https://user-images.githubusercontent.com/154725/177942440-77d7e69c-2e1f-4227-a19e-7302b2ed050f.png">

Reply:
Yeah, it should be doable. Do you want to do it here or in a separate PR?

Reply:
I was thinking we should do it before we merge, but I guess that shouldn't really matter, should it?

Reply:
It's not that many, so I can just as well do it now, and we can get it tested before merging.

Reply:
Great! :slightly_smiling_face: